### ISSUE TITLE
issue about DSC metric

### ISSUE THREAD

**Issue by diliess** [usagisukisuki/Adaptive_t-vMF_Dice_loss#1]
I would like to know why the effect of your code on the CVC_clinic data set after training is only 0.84 in the test set DSC coefficient, but it can reach 0.93 on the verification set, why is there such a big gap， Also, why does the FCBformer paper show that the DSC is 0.94 when DICE and CE losses are added together as a loss function, while it is 0.90 in your paper? thank you！
